### DISCUSSION TITLE
Is expo-status-bar still required?

### DISCUSSION THREAD

**Discussion by GollyJer** [zoontek/react-native-edge-to-edge#36] [Q&A] [answered]
I uninstalled expo-status-bar and ripped out it's code, then added react-native-edge-to-edge and did a new build.
On first run of the app I get 
```Unable to resolve "expo-status-bar" from "node_modules\expo-router\build\ExpoRoot.js"```

I'm currently doing a new build with both packages installed.
Is this to be expected or did I maybe do something wrong?

**Comment by zoontek** [comment, marked as answer]
@GollyJer Yes, but this is not due to this package. Current version of Expo router embed `expo-status-bar` as a dependency, but it's nearly useless.

[They removed it](https://github.com/expo/expo/commit/8e9fcaad45df82892c436cd9d3ad95e378a2cf80#diff-999071a0dfb507af3e9096ca92f42027e98aa2a51d970a4fa955d069ea50447dL55), and this change will probably be published alongside Expo 52.

For the moment, keep it, but don't use it (use `SystemBars`). Soon you will be able to update and drop the dependency too.
